devicon.json

```diff
@@ -2512,6 +2512,25 @@
         "color": "#0073BA",
         "aliases": []
     },
+    {
+        "name": "dynamodb",
+        "altnames": [],
+        "tags": [
+            "key-value",
+            "database"
```

Comment:
```suggestion
            "database",
            "java"
```

Reply:
why adding java though?

Reply:
I believe this tool/framework uses Java. 🤔

Reply:
I didn't find any other databases that had their built language added with them, `postgresql` didn't have `c++` nor did `mysql` have `C++`. Do we need to add it here?

Reply:
We don't have to, but if it's related, it's best to add them.
Also, those are icons added quite a long time ago, so a few things have changed.

Reply:
okay cool, I'll add it now.